examples/simulated/README.txt

```diff
@@ -0,0 +1,4 @@
+Examples with simulated data
```

Comment:
To be homogeneous with other titles of the gallery, I would name it simply "Simulated data".